### ISSUE TITLE
Move tests folder from download folder

### ISSUE THREAD

**Issue by CrafterKolyan** [choldgraf/download#32]
If you put `tests` folder in `download` folder then it means that EVERY user installed this package has these files in package folder.

**Comment by CrafterKolyan** [comment]
This is actually not true. `tests` folder is not in `download` folder in `site-packages` of python

**Comment by choldgraf** [comment]
yeah I believe that folders that aren't part of the python module need to be explicitly included in the package if you want them when you in stall with `pip`
